x/projects/client/cli/tx_add_project_keys.go

```diff
@@ -14,22 +15,34 @@ var _ = strconv.Itoa(0)
 
 func CmdAddProjectKeys() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "add-project-keys",
-		Short: "Broadcast message AddProjectKeys",
-		Args:  cobra.ExactArgs(0),
+		Use:   "add-project-keys [project-id] [project-keys-file-path]",
+		Short: "Add developer/admin keys to an existing project",
+		Long: `The add-project-keys command allows the project admin to add new project keys (admin/developer) to the project.
+		To add the keys you need to provide a YAML file of the new project keys (see example in cookbook/project/example_project_keys.yml).
+		Note that in project keys, to define the key type, you should follow the enum described in the top of example_project_keys.yml.`,
+		Example: `required flags: --from <admin-key> (the project's subscription address is also considered admin)
+				  
+		lavad tx subscription add-project-keys [project-id] [project-keys-file-path] --from <admin-key>`,
+		Args: cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			clientCtx, err := client.GetClientTxContext(cmd)
+			projectID := args[0]
```

Comment:
lets change this cobra.ExactArgs(2), -> cobra.RangeArge(1,2),
yaml will be optional
also the user can use flags such as --developer or --admin to add keys, in repeted flags like this:
add-project-keys --developer lava@one --developer lava@two --developer lava@three